internal/service/federatedsettingsorgconfig/resource_federated_settings_connected_org_test.go

```diff
@@ -95,10 +94,11 @@ func importStateIDFunc(resourceName, federationSettingsID, orgID string) resourc
 func configBasic(federationSettingsID, orgID, identityProviderID string) string {
 	return fmt.Sprintf(`
 	resource "mongodbatlas_federated_settings_org_config" "test" {
-		federation_settings_id = "%[1]s"
-		org_id                 = "%[2]s"
-		domain_restriction_enabled = false
-		domain_allow_list          = ["reorganizeyourworld.com"]
-		identity_provider_id = "%[3]s"
+		federation_settings_id 		= %[1]q
+		org_id                 		= %[2]q
+		domain_restriction_enabled 	= false
+		domain_allow_list          	= ["mydomain.com"]
```

Comment:
seeing this test data change it looks like we're not checking that param in the tests

Reply:
good catch @lantoli! Took two actions:
1. restored to original values (I was making some tests initially because it wasn't working).
2. Added more test checks to cover the other attributes.
Let me know if this works for you.